tests/drivers/charger/sbs_charger/src/test_sbs_charger.c

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
note: you can create fake drivers (using FFF) and create much more flexible (and faster) tests, see e.g. https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/drivers/regulator/api/src/main.c. The platform hackery in testcase.yaml could likely be avoided.